src/coreclr/vm/eetwain.cpp

```diff
@@ -2176,9 +2176,25 @@ DWORD_PTR InterpreterCodeManager::CallFunclet(OBJECTREF throwable, void* pHandle
 
     StackVal retVal;
 
-    frames.interpMethodContextFrame.startIp = pOriginalFrame->startIp;
-    frames.interpMethodContextFrame.pStack = isFilter ? sp : pOriginalFrame->pStack;
-    frames.interpMethodContextFrame.pRetVal = (int8_t*)&retVal;
+#ifndef FEATURE_REUSE_INTERPRETER_STACK_FOR_NORMAL_FUNCLETS
+    // Frame pointer for original method
+    int8_t* originalStack;
+    if (pOriginalFrame->IsFuncletFrame())
+    {
+        originalStack = *(int8_t**)pOriginalFrame->pStack;
```

Comment:
Just checking: It's a funclet frame, so we reserved the first sizeof(void*) bytes of the funclet frame's stack to contain the address of the original frame's stack, and we're extracting it from there.